README.md

```diff
@@ -1,3 +1,12 @@
-# Worker for running the regro-cf-autotick-bot 
+# Worker for running the regro-cf-autotick-bot
 
 [![CircleCI](https://circleci.com/gh/regro/circle_worker.svg?style=svg)](https://circleci.com/gh/regro/circle_worker)
+
+
+# Starting and Stopping the Worker
+
+In order to start the worker, make a commit to master with the file `please.go`
+in the top-level directory.
+
+If you want to stop the worker, simply delete this file and it will not restart
```

Comment:
Do we want checks inside the potentially damaging parts of the code (auto_tick.py) that checks that this exists, in case we want to directly halt a long run?

Reply:
We could add those later. As I said earlier, my preference here would be to not run the bot a lot longer at first.